README.md

```diff
@@ -1 +1,157 @@
-# terraform-aws-billing
\ No newline at end of file
+# Terraform Module: terraform-aws-billing
+# This module facilitates the creation of AWS Budgets with flexible notification options, including email, Slack, and Google Chat.
+
+## Overview
+The `terarform-aws-billing` module enables you to easily deploy AWS Budgets, helping you monitor and control your AWS spending. The module supports multiple notification channels, allowing you to receive alerts through email, Slack, and Google Chat when your account budget thresholds are reached.
+
+## Usage
+```hcl
+//Configuration for both email and Google chat webhook
+module billing {
+  source = "github.com/dedicatted/terraform-aws-billing"
+  subscriber_email_addresses = ["test@gmail.com"]
+  google_chat_webhook = "/v1/spaces/space/messages?key=key&token=token"
+  google_chat_webhook_enabled = true
+}
+```
+```hcl
+//Configuration for Google chat webhook
+module billing {
+  source = "github.com/dedicatted/terraform-aws-billing"
+  subscriber_email_addresses = []
```

Comment:
need to delete this string if it has the same default value

Reply:
done